lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -40,6 +51,20 @@ class AnnotationDriver extends AbstractAnnotationDriver
         Mapping\MappedSuperclass::class => 2,
     ];
 
+    /**
+     * Initializes a new AnnotationDriver that uses the given AnnotationReader for reading
+     * docblock annotations.
+     *
+     * @param Reader               $reader The AnnotationReader to use, duck-typed.
```

Comment:
We should move away from the `duck-typed` part, imho. It has caused us trouble already.